client/css/style.css

```diff
@@ -128,7 +128,7 @@ button,
 	border-radius: 3px;
 	color: #84ce88;
 	display: inline-block;
-	font: bold 12px Lato, sans-serif;
+	font: bold 12px;
```

Comment:
This is not a valid declaration. It is ignored entirely by Chrome. I think you will have to either use `font: bold 12px inherit` or split it into two lines.